input/pagecontent/reference.md

```diff
@@ -1682,13 +1700,23 @@ Creating a code system uses the keywords `CodeSystem`, `Id`, `Title` and `Descri
 * #{code} "{display string}" "{definition string}"
 ```
 
+Codes that are contained within other codes can also be defined. The result is a hierarchical structure of codes within a code system. To define such codes, list all of the preceding codes in the hierarchy before the new code:
```

Comment:
According to the FHIR spec, "the nature of the relationships is variable (is-a/contains/categorizes)" -- so we should avoid the word "contained" here since it may be too narrow.  I'd suggest something more like:
> Child codes can also be defined, resulting in a hierarchical structure of codes within a code system.  To define such codes...